ddtrace/contrib/django/middleware.py

```diff
@@ -57,10 +56,7 @@ def process_response(self, request, response):
             span = _get_req_span(request)
             if span:
                 span.set_tag(http.STATUS_CODE, response.status_code)
-
-                if apps.is_installed("django.contrib.auth"):
-                    span = _set_auth_tags(span, request)
-
+                span = _set_auth_tags(span, request)
```

Comment:
Cool! @mattrobenolt was it the only problem for Django < 1.7? Anyway, it's good to go!